### ISSUE TITLE
Extend \ifusexxx to each type of name

### ISSUE THREAD

**Issue by maieul** [plk/biblatex#277]
Hi,

actually, biblatex provides options useeditor, useauthor, usetranslator which : 
- switches the result of \ifuseeditor, \ifuseauthor, \ifusetranslator etc.
- disable the field in the \DeclareLabelName and in the sorting schemes.

I have declared a new field, called "realauthor". Maybe I have make a mistake in the reading of the handbook, but I have found no way to create a similar command/option userealauthor.

**Comment by plk** [comment]
Could you try this and see? I would like to release biblatex 3.0 soon and this is one of the new features.

**Comment by maieul** [comment]
I have tried biber 3.0, but I obtained 

Can't locate List/MoreUtils/PP.pm in @INC (you may need to install the List::MoreUtils::PP module) (@INC contains: CODE(0x1014ffa30) /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc CODE(0x101231120) CODE(0x10123c1f8)) at (eval 48) line 1.

etc etc.

maybe I have token the wrong ? I used http://sourceforge.net/projects/biblatex-biber/files/biblatex-biber/development/binaries/OSX_Intel/

**Comment by maieul** [comment]
the full message
`````
Can't locate List/MoreUtils/PP.pm in @INC (you may need to install the List::MoreUtils::PP module) (@INC contains: CODE(0x1014ffa30) /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc CODE(0x101231120) CODE(0x10123c1f8)) at (eval 48) line 1.
BEGIN failed--compilation aborted at (eval 48) line 1.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils/XS.pm line 41.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils.pm line 13.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils.pm line 13.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/AllUtils.pm line 13.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/AllUtils.pm line 13.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Config.pm line 15.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Config.pm line 15.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/LaTeX/Recode.pm line 6.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/LaTeX/Recode.pm line 6.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Utils.pm line 20.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Utils.pm line 20.
Compilation failed in require at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Internals.pm line 8.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Internals.pm line 8.
Compilation failed in require at /loader/HASH(0x1008610f8)/base.pm line 99.
	...propagated at /loader/HASH(0x1008610f8)/base.pm line 108.
BEGIN failed--compilation aborted at /var/tmp/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber.pm line 5.
Compilation failed in require at script/biber-darwin line 20.
BEGIN failed--compilation aborted at script/biber-darwin line 20.
maieul@FTSR22998:documentation (master %)$ ./biber *bcf
Can't locate List/MoreUtils/PP.pm in @INC (you may need to install the List::MoreUtils::PP module) (@INC contains: CODE(0x1014ffa18) /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc CODE(0x101231120) CODE(0x10123c1f8)) at (eval 48) line 1.
BEGIN failed--compilation aborted at (eval 48) line 1.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils/XS.pm line 41.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils.pm line 13.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/MoreUtils.pm line 13.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/AllUtils.pm line 13.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/List/AllUtils.pm line 13.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Config.pm line 15.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Config.pm line 15.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/LaTeX/Recode.pm line 6.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/LaTeX/Recode.pm line 6.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Utils.pm line 20.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Utils.pm line 20.
Compilation failed in require at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Internals.pm line 8.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber/Internals.pm line 8.
Compilation failed in require at /loader/HASH(0x1008610f8)/base.pm line 99.
	...propagated at /loader/HASH(0x1008610f8)/base.pm line 108.
BEGIN failed--compilation aborted at /var/folders/0g/vh5y8yv93js6v76gmjpnt4mr0000gr/T/par-6d616965756c/cache-560c6f4a0f9251f626b033329df359db7c50e7cc/inc/lib/Biber.pm line 5.
Compilation failed in require at script/biber-darwin line 20.
BEGIN failed--compilation aborted at script/biber-darwin line 20.
````

**Comment by plk** [comment]
Hmm, strange, I can reproduce it, looking into it.

**Comment by plk** [comment]
Should be fixed if you pull biber 2.0 again.

**Comment by maieul** [comment]
all is working ! thanks.
